.github/workflows/run_tests.yml

```diff
@@ -25,7 +25,7 @@ jobs:
         run: vendor/bin/phpunit
 
       - name: Run phpcs
-        run: vendor/bin/phpcs --standard=PSR12 --exclude=Generic.Files.LineLength ./src
+        run: composer run test:cs
```

Comment:
We can directly write `composer test:cs`, do not add `run`, It would be better if you can move the `phpstan` to composer scripts

Reply:
Being implicit makes us not being explicit. Which is bad.